src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -0,0 +1,42 @@
+export interface ApiKeyModel {

```

Comment:
You can find better type definitions from the swagger
![image](https://user-images.githubusercontent.com/35579930/66870365-4d5f2480-ef6f-11e9-85ef-3ee2ae4cd134.png)

Reply:
Everything is optional in this model